cli/src/cmd/utils.rs

```diff
@@ -228,3 +230,33 @@ pub fn has_batch_support(chain: u64) -> bool {
     }
     true
 }
+
+pub trait LoadConfig {
+    fn load_config_emit_warnings(self) -> Config;
```

Comment:
let's also add a regular `load_config` function here, this makes it easy to explain the difference